docs/src/api/class-android.md

```diff
@@ -3,6 +3,21 @@
 
 Playwright has **experimental** support for Android automation. See [here](./mobile.md) for more information. You can access android namespace via:
 
+*Requirements*
```

Comment:
now its quite similar to https://playwright.dev/docs/mobile

can we nuke the mobile page?

Reply:
Yes, we should, our mobile page is not indexed...